prog/weave-kube/launch.sh

```diff
@@ -54,6 +54,12 @@ else
     fi
 fi
 
+STATUS_OPTS="--metrics-addr=$METRICS_ADDR"
+# --status-addr exposes internal information, so only turn it on if asked to.
```

Comment:
1. `WEAVE_STATUS_ADDR` and `WEAVE_METRICS_ADDR` should probably be mentioned in `site/kubernetes/kube-addon.md`.
2. `WEAVE_METRICS_ADDR` in `site/tasks/manage/metrics.md`.